serde/json.py

```diff
@@ -1,50 +1,73 @@
 """
 Serialize and Deserialize in JSON format.
 """
-import json
-from typing import Any, Type
+from typing import Any, Optional, Type, Union
 
 from .compat import T
 from .de import Deserializer, from_dict
 from .numpy import encode_numpy
 from .se import Serializer, to_dict
 
+try:  # pragma: no cover
+    import orjson
+
+    def json_dumps(obj, **opts) -> str:
+        if "option" not in opts:
+            opts["option"] = orjson.OPT_SERIALIZE_NUMPY
+        return orjson.dumps(obj, **opts).decode()
+
+    def json_loads(s, **opts) -> Any:
+        return orjson.loads(s, **opts)
+
+except ImportError:
+    import json
+
+    def json_dumps(obj, **opts):
+        if "default" not in opts:
+            opts["default"] = encode_numpy
+        # compact output
+        return json.dumps(obj, ensure_ascii=False, separators=(",", ":"), **opts)
```

Comment:
Could you fix to allow users to pass own `ensure_ascii` and `separators` variables in `to_json` like this? 

```python
ensure_ascii = opts.pop("ensure_ascii", False)
separators = opts.pop("separators", (",", ":"))
return json.dumps(obj, ensure_ascii=ensure_ascii, separators=separators, **opts)
```

I got this error without it
```
TypeError: json.dumps() got multiple values for keyword argument 'separators
```